upgrade/scripts/upgrade/util/upgrade-test-rpms.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ncns=$(grep -oP 'ncn-\w\d+' /etc/hosts | sort -u |  tr -t '\n' ',')
+
+echo "Installing updated versions of csm-testing goss-servers craycli rpms"
+pdsh -b -w ${ncns} 'zypper install -y csm-testing goss-servers craycli'
+
+echo "Enabling and starting goss-servers"
+pdsh -b -w ${ncns} 'systemctl enable goss-servers && systemctl start goss-servers'
```

Comment:
Enable `pdsh` to fail on a non-zero return code with `-S`.

```suggestion
pdsh -S -b -w ${ncns} 'zypper install -y csm-testing goss-servers craycli'

echo "Enabling and starting goss-servers"
pdsh -S -b -w ${ncns} 'systemctl enable goss-servers && systemctl start goss-servers'
```